dll/win32/shell32/res/rgs/copyaspathmenu.rgs

```diff
@@ -0,0 +1,28 @@
+HKCR
+{
+	NoRemove CLSID
```

Comment:
Please use 4-space indentation.

Reply:
Those rgs files all seem to use tabs.

Reply:
Maybe the existing ones are not optimal, old, and maybe partially be inherited from Wine, yes. But yours is an entirely new file. So the proper project rules should be applicable here.